note that deferred callback can only have ephemeral as flag (#7386)
hammerandchisel/discord-api-docs: docs/interactions/Receiving_and_Responding.mdx

| components?       | array of [components](#DOCS_INTERACTIONS_MESSAGE_COMPONENTS/)                    | Message components                                                                                                                                                                                                     |
| attachments? \*\* | array of partial [attachment](#DOCS_RESOURCES_MESSAGE/attachment-object) objects | Attachment objects with filename and description                                                                                                                                                                       |
| poll?             | [poll](#DOCS_RESOURCES_POLL/poll-create-request-object) request object           | Details about the poll                                                                                                                                                                                                 |

> \* If you create a callback with the [type](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-response-object-interaction-callback-type) `DEFERRED_CHANNEL_MESSAGE_WITH_SOURCE` you can only set `EPHEMERAL` as flag.

Reviewer: ```suggestion
\* If you create a callback with the [type](#DOCS_INTERACTIONS_RECEIVING_AND_RESPONDING/interaction-response-object-interaction-callback-type) `DEFERRED_CHANNEL_MESSAGE_WITH_SOURCE` the only valid [message flag](#DOCS_RESOURCES_MESSAGE/message-object-message-flags) you may use is `EPHEMERAL`.
```
